packages/base-map/package.json

```diff
@@ -9,20 +9,22 @@
   "module": "esm/index.js",
   "private": false,
   "dependencies": {
-    "prop-types": "^15.7.2"
+    "@opentripplanner/types": "^2.0.0",
```

Comment:
Could you use types 3.0.0 please? Oh, it is a dev dependency.

Reply:
Good call thanks have updated